pom.xml

```diff
@@ -33,64 +33,79 @@
       <groupId>com.atlassian.bitbucket.server</groupId>
       <artifactId>bitbucket-api</artifactId>
       <scope>provided</scope>
-      <version>4.11.1</version>
     </dependency>
     <dependency>
       <groupId>com.atlassian.bitbucket.server</groupId>
-      <artifactId>bitbucket-spi</artifactId>
+      <artifactId>bitbucket-build-api</artifactId>
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.atlassian.sal</groupId>
-      <artifactId>sal-api</artifactId>
+      <groupId>com.atlassian.bitbucket.server</groupId>
+      <artifactId>bitbucket-spi</artifactId>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.atlassian.bitbucket.server</groupId>
-      <artifactId>bitbucket-build-api</artifactId>
+      <artifactId>bitbucket-util</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.atlassian.plugins.rest</groupId>
+      <artifactId>atlassian-rest-common</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.atlassian.sal</groupId>
+      <artifactId>sal-api</artifactId>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.atlassian.templaterenderer</groupId>
       <artifactId>atlassian-template-renderer-api</artifactId>
-      <version>1.5.7</version>
```

Comment:
In general, if you have a `<scope>import</scope>` for `bitbucket-parent`, you shouldn't define explicit versions on your dependencies. Our parent POM defines the versions of the libraries that we actually ship with, so using those versions ensures you're (for example) using the "right" APIs for Bitbucket Server versions you target.

Reply:
Thanks for the tip. The POM was largely copied from Atlassian's example plugin project, so if you haven't already you might want to update that to prevent others from doing this.